### PR TITLE
feat: upgrade metrics server

### PR DESCRIPTION
**Pull request by MacLikorne**

### CODE REVIEW

File: src/cloud_provider/digitalocean/kubernetes/helm_charts.rs [hunk 1]
```diff
@@ -428,15 +428,8 @@ pub fn do_helm_charts(
         chart_info: ChartInfo {
             name: "metrics-server".to_string(),
             path: chart_path("common/charts/metrics-server"),
+            values_files: vec![chart_path("chart_values/metrics-server.yaml")],
             values: vec![
-                ChartSetValue {
```
Comment (deimosfr) [inline src/cloud_provider/digitalocean/kubernetes/helm_charts.rs:432]: why did you remove it?

File: src/cmd/kubectl.rs [hunk 2]
```diff
@@ -1239,3 +1239,14 @@ where
         envs,
     )
 }
+
+pub fn kubernetes_get_all_hpas<P>(kubernetes_config: P, envs: Vec<(&str, &str)>) -> Result<HPA, SimpleError>
```
Comment (deimosfr) [inline src/cmd/kubectl.rs:1243]: Please take a look on how other similar cmd works to select namespaces. It would be simpler for later usage in a single namespace. Thx

File: test_utilities/src/common.rs [hunk 3]
```diff
@@ -1510,3 +1544,34 @@ pub fn cluster_test(
...
+where
+    P: AsRef<Path>,
+{
+    let result = kubernetes_get_all_hpas(kubernetes_config, envs);
+
+    match result {
+        Ok(hpas) => {
+            for hpa in hpas.items.expect("No hpa item").into_iter() {
+                if hpa.metadata.annotations.is_some() {
+                    if hpa
```
Comment (deimosfr) [inline test_utilities/src/common.rs:1558]: can't you check something working? because if it's another error, we may have issues

File: src/cloud_provider/digitalocean/kubernetes/helm_charts.rs (hunk 1 above)
Comment (MacLikorne) [reply src/cloud_provider/digitalocean/kubernetes/helm_charts.rs:432]: Because it was inconsistent. Replaced for all by a values file. If you prefer I can put it all in rust instead of yaml.

**Review by deimosfr — APPROVED** [review]
